.gitignore

```diff
@@ -1,7 +1,12 @@
 *.os
 *.so
 *.cfgc
+*.cfg
 *~
 *.pyc
 utilLib.py
 *_wrap.cc
+*.log
+.scons*
+conftest*
+ups_db
```

Comment:
This file needs a final newline.  Why do you need to add ups_db here?   Do you have your eochar repo directory in your EUPS_PATH environment variable?  I'm not sure what the "right" path entry would be, but you can definitely have the ups_db directory be somewhere else.